Source/mpqapi.cpp

```diff
@@ -103,25 +103,21 @@ void __fastcall mpqapi_update_multi_creation_time(DWORD dwChar)
 
 BOOLEAN __fastcall mpqapi_reg_store_modification_time(char *pbData, int dwLen)
 {
-	int v2;          // ebx
-	char *v3;        // ebp
-	char *v4;        // edi
-	unsigned int v5; // esi
-	char *v6;        // ecx
+	char *pbCurrentData;
+	unsigned int i;

```

Comment:
Is it still bin exact if you change it to `DWORD`?

Reply:
Yep, I will modify the PR.